python/setup.py

```diff
@@ -204,13 +190,6 @@ def write_version_py(filename=VERSION_PY_PATH):
         ext_modules=ext_modules,
         scripts=['bin/sbp2json'],
         extras_require={
-          'jit': [
-              'numpy~=1.18',
-              'pybase64~=0.3',
-              'python-rapidjson~=1.0',
-              'numba==0.47',
-              'llvmlite==0.31',
-          ],
           'aot': [
```

Comment:
we need to remove/transform the `aot` extra too since it was essentially just the "pre-compiled" variant of sbp.jit-- should probably instead transform this to `sbp2json`